libp2p/8-implementations.md

```diff
@@ -1,40 +1,57 @@
 8 Implementations
 =================
 
-This is a list of known libp2p module implementations. They are components that respect the interfaces and expectations defined in the "Interfaces" chapter, and can be composed to make a working libp2p library.
+An libp2p implementation should(recommended) follow a certain level of granulatiry when implementing different modules and functionalities, so that common interfaces are easy to expose, test and check for interoperability with other implementations.
+
+This is the list of current modules available for libp2p:
+
+- libp2p (entry point)
```

Comment:
I remember having this conversation and agreeing with you, should have added it as a TODO right away.

I like `interface-X`.

:+1: for Haskell too, however, we will have a mapping for each language as implementations appear :)